src/common/telemetry.py

```diff
@@ -27,15 +17,13 @@ def send_telemetry(self, event_name):
             and not self.telemetry_state[event_name]
         ):
             self.telemetry_client.track_event(
-                "{}/{}".format(
-                    self.extension_name, CONSTANTS.TELEMETRY_EVENT_NAMES[event_name]
-                )
+                f"{self.extension_name}/{CONSTANTS.TELEMETRY_EVENT_NAMES[event_name]}"
```

Comment:
Why are we passing a string inside a function to fetch the telemetry constant. It would be easier by passing the constant directly. ex:send_telemtry(CONSTANTS.TEMPERATURE)

Reply:
agreed. made an enum for it